.github/workflows/largemodel_unit_test_CI.yml

```diff
@@ -150,6 +150,8 @@ jobs:
           echo "Deleted vespa_tester_app.zip"
 
       - name: Run Large Model Unit Tests
+        id: run_unit_tests
+        continue-on-error: true
```

Comment:
is this required so that even on failure the test coverage report is generated?

Reply:
yes, This is to make sure we proceed to following steps even if this step fails. for example, even the test fails, we still generate and upload coverage report and test report.